internal/server/v1beta1/search.go

```diff
@@ -135,3 +136,13 @@ func filterConfigFromValues(fltMap map[string]string) map[string][]string {
 	}
 	return filter
 }
+
+func getSearchFlagsFromFlags(inputFlags *compassv1beta1.SearchFlags) *asset.SearchFlags {
+	if inputFlags != nil {
+		return &asset.SearchFlags{
+			EnableHighlight: inputFlags.EnableHighlight,
+		}
+	} else {
+		return nil
+	}
+}
```

Comment:
```go
func getSearchFlagsFromFlags(inputFlags *compassv1beta1.SearchFlags) asset.SearchFlags {
	return asset.SearchFlags{
		EnableHighlight: inputFlags.GetEnableHighlight(),
	}
}
```

`inputFlags.GetEnableHighlight()` has a `nil` check so it is safe to call on a `nil` instance.

Reply:
Done